### PR TITLE
Clarified readme file

### PR DESCRIPTION
**Pull request by DovydasNavickas**
Just a clarification for people looking to start fast.

/cc @PinpointTownes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,6 +97,14 @@ app.UseOpenIdConnectServer(options => {
 });
 ```
 
+Also, remember to add authentication services:
```
Comment (PinpointTownes) [inline README.md]: How about: `In order for the OpenID Connect server middleware to work properly, the authentication services must be registered in the DI container`?

File: README.md [hunk 2]
```diff
@@ -97,6 +97,14 @@ app.UseOpenIdConnectServer(options => {
...
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
```
Comment (PinpointTownes) [inline README.md]: More K&R love please :smile:

**Comment by DovydasNavickas** [comment]
Updated :)

**Comment by PinpointTownes** [comment]
Looks fine, thanks! Any chance you could squash your PR into a single commit? :smile:

**Comment by DovydasNavickas** [comment]
Doesn't git let you do that automatically now?

I will though.

**Comment by PinpointTownes** [comment]
> Doesn't git let you do that automatically now?

I'm an old school guy, you know :smile: 

Merged, thanks! https://github.com/aspnet-contrib/AspNet.Security.OpenIdConnect.Server/commit/a07a83600e6ca7727fa272df4c904496bbbbd8c8 :clap:

**Pull request closed without merging**